client/src/pages/home/HomePage.tsx

```diff
@@ -0,0 +1,10 @@
+import Sidebar from '../../components/Layout/Sidebar';
+
+export const HomePage = () => {
+  return (
+    <>
+      <Sidebar />
```

Comment:
It's necessary that Sidebar is visible throughout the whole app. Please add that feature. [Docs](https://reactrouter.com/en/main/start/tutorial#nested-routes) shows great example on how to do that.